config/test.exs

```diff
@@ -4,8 +4,8 @@ config :logger, :console,
   format: "\n$date $time [$level]: $message \n"
 
 config :exq,
-  host: String.to_char_list(System.get_env("REDIS_HOST") || "127.0.0.1"),
```

Comment:
So this no longer supports the env variable? Seems that it would still work with this old code if this wasn't set.

Reply:
`String.to_char_list` cannot be used with newer elixir versions, that's why i removed it, plus the fact that no env var were set on travis. If in the future env var are needed, just keep `System.get_env("REDIS_HOST")` as the value will be cast as a string